### ISSUE TITLE
v2.25.1 slow fetching speed

### ISSUE THREAD

**Issue by giiutfff** [SmartRSS/Smart-RSS#225]
When using v2.23 updating the feed list takes a few seconds. Now after updated to v2.25.1 emmmmmm maybe I should fix myself a cup of tea then come back to check the result? Anyway it fixed issues in v2.25 at least (everything now can be fetched, after some waiting). But should this to be considered a new regression? 
btw, I only update manually.

**Comment by zakius** [comment]
Do you use a lot of feeds on the same domain? I intentionally throttle connections to same domain to reduce the load, but possibly this can be finetuned a bit to make it less annoying when using big services.

**Comment by giiutfff** [comment]
Yes, like a few dozens from the same website. But v2.23 can update them so fast. Is this some change to meet the new requirement set by Google or other companies? 
Also, start form second feed the loading animating icon is not displayed correctly (showed for less than a second then disappeared)

**Comment by zakius** [comment]
This is a new change, it isn't really required for all services, but some misbehave when flooded like that and that's a way to prevent that. I'll try tuning it to be less annoying, but it really should've been there from the very beginning. I'll take a look at the animation too
